### ISSUE TITLE
Cannot share file due to error reading ACL

### ISSUE THREAD

**Issue by peter279k** [solid/node-solid-server#1236]
# Instruction
- I use the master branch with latest commits to try building the Solid server.
- Firstly, I create a Solid server with `Dockerfile`.
- It looks fine and I also create a new Pod server named `https://lee.electric-data-pod.local:8443`. The related snapshots are as follows:

![image](https://user-images.githubusercontent.com/9021747/59748811-77992200-92ae-11e9-90a5-68f9ce116827.png)

- I also create a `private_folder` folder and a `test.txt` successfully.

![image](https://user-images.githubusercontent.com/9021747/59748917-a911ed80-92ae-11e9-94c2-cf8f8c1bd4f1.png)

- When I try setting sharing file setting, I got the following snapshot:

![image](https://user-images.githubusercontent.com/9021747/59748996-cb0b7000-92ae-11e9-9dc1-5d2e6f1884ee.png)

It seems that I don't have the `ACL` file for this `test.txt` and I notice that the access control file will not be created when I create the `test.txt` file.

What should I solve this problem? Creating a `ACL` file for `test.txt` manually?

Is there any way to create related `ACL` file for specific files automatically?

Thanks.

**Comment by jaxoncreed** [comment]
You're right, it should generate an acl file automatically. Let me see if I can reproduce this.

**Comment by peter279k** [comment]
Hi @jaxoncreed, thank you for your help.

**Comment by michielbdejong** [comment]
Can you share your config.json? Is https://electric-data-pod.local:8443 your serverUri and 'lee' your username, running in multi-user mode?

**Comment by peter279k** [comment]
Hi @michielbdejong, thank you for your concern. My `config.json` is as follows:

```
{
  "root": "./data",
  "port": "8443",
  "serverUri": "https://electric-data-pod.local:8443",
  "webid": true,
  "mount": "/",
  "configPath": "./config",
  "dbPath": "./.db",
  "sslKey": "./privkey.pem",
  "sslCert": "./fullchain.pem",
  "multiuser": true,
  "corsProxy": "/proxy",
  "server": {
    "name": "electric-node-solid-server",
    "description": "This is the forked node-solid-server for the electric data pod",
    "logo": ""
  },
  "enforceToc": true,
  "disablePasswordChecks": false,
  "tocUri": "https://your-toc",
  "supportEmail": "peter279k@gmail.com",
  "database": {
    "development": {
      "mysqlHost": "",
      "mysqlDBName": "",
      "mysqlDBUser": "",
      "mysqlDBPassword": ""
    },
    "production": {
      "mysqlHost": "",
      "mysqlDBName": "",
      "mysqlDBUser": "",
      "mysqlDBPassword": ""
    }
  }
}
```

BTW, just ignore the `database` block, this block is for my customized database adapter.
